tests/framework/microvm.py

```diff
@@ -151,6 +151,9 @@ def __init__(
         # Flag checked in destructor to see abnormal signal-induced crashes.
         self.expect_kill_by_signal = False
 
+        # MMDS content from file
+        self._metadata_file = ''
```

Comment:
Why not have this be `None` or something else?

Reply:
No reason, i have changed this.